static/js/map.js

```diff
@@ -840,6 +842,11 @@ function customizePokemonMarker(marker, item, skipNotification) {
                 marker.setAnimation(google.maps.Animation.BOUNCE)
             }
         }
+        if (Store.get('remember_show_ivzero') && perfection === 0) {
```

Comment:
Why use Store.get when you are already storing the value in $selectnotifiyIvZero